pkg/cloud/services/networking/securitygroups.go

```diff
@@ -321,7 +321,7 @@ func (s *Service) createSecurityGroupIfNotExists(openStackCluster *infrav1.OpenS
 		return err
 	}
 	if secGroup == nil || secGroup.ID == "" {
-		s.scope.Logger().V(6).Info("Group doesn't exist, creating it.", "name", groupName)
+		s.scope.Logger().V(6).Info("Group doesn't exist, creating it", "name", groupName)
```

Comment:
Somewhat unrelated, but I assume there are no issues with using contractions (`doesn't` rather than `does not`) in logs?

Reply:
Yeah, the guideline didn't mention the contraction issue.